app/controllers/foreman_tasks/recurring_logics_controller.rb

```diff
@@ -1,13 +1,23 @@
 module ForemanTasks
   class RecurringLogicsController < ::ApplicationController
-    before_action :find_recurring_logic, :only => [:show, :cancel]
+    before_action :find_recurring_logic, :only => [:show, :cancel, :enable, :disable]
 
     def index
       @recurring_logics = filter(resource_base)
     end
 
     def show; end
 
+    def enable
+      @recurring_logic.update!(:enabled => true)
+      redirect_to :action => :index
+    end
+
+    def disable
+      @recurring_logic.update!(:enabled => false)
+      redirect_to :action => :index
```

Comment:
Clicking disable/enable when looking at recurring logic details redirects you to the index. I think the it should stay on the same page, but on the other hand cancel behaves the same way. Guess we can be consistent first and fix this behavior in another PR